src/coreclr/src/vm/eventpipe.cpp

```diff
@@ -40,6 +40,9 @@ unsigned int * EventPipe::s_pProcGroupOffsets = nullptr;
 #endif
 Volatile<uint32_t> EventPipe::s_numberOfSessions(0);
 bool EventPipe::s_enableSampleProfilerAtStartup = false;
+CLREventStatic *EventPipe::s_ResumeRuntimeStartupEvent = new CLREventStatic();
```

Comment:
Please do not run any significant code in static constructors. It is a pit of failure.